frontend/src/features/admin-form/settings/mutations.ts

```diff
@@ -138,6 +142,54 @@ export const useMutateFormSettings = () => {
     },
   )
 
+  const mutateFormHasMultiLang = useMutation(
+    (nextHasMultiLang: boolean) =>
+      updateFormHasMultiLang(formId, nextHasMultiLang),
+    {
+      onSuccess: (newData) => {
+        handleSuccess({
+          newData,
+          toastDescription: newData.hasMultiLang
+            ? 'Multi-language enabled. Respondents can now select other languages to view your form in.'
+            : 'Multi-language disabled.',
+        })
+      },
+    },
+  )
+
+  const mutateFormSupportedLanguages = useMutation(
+    (nextSupportedLanguages?: FormSupportedLanguages) =>
+      updateFormSupportedLanguages(
+        formId,
+        nextSupportedLanguages?.nextSupportedLanguages,
+      ),
+    {
+      onSuccess: (newData, newSupportedLanguages) => {
+        if (newSupportedLanguages && newSupportedLanguages.selectedLanguage) {
+          const supportedLanguages =
+            newSupportedLanguages.nextSupportedLanguages ?? []
+          const languageToDisplay = convertUnicodeLocaleToLanguage(
+            newSupportedLanguages.selectedLanguage,
+          )
+
+          let successMessage: string
```

Comment:
nit: prefer using ternary operator with a const successMessage instead of let

the rationale is that using const as much as possible reduces the likelihood of undesired mutations in the code. 

```suggestion
          const isSelectedLanguageSupported = supportedLanguages.includes(newSupportedLanguages.selectedLanguage)
          const successMessage = isSelectedLanguageSupported ? <str1> : <str2>; 
```